clinvar_ingest/cloud/bigquery/create_tables.py

```diff
@@ -49,7 +50,8 @@ def schema_file_path_for_table(table_name: str) -> str:
     """
     Returns the path to the BigQuery schema file for the given table name.
     """
-    schema_path = bq_schemas_dir / f"{table_name}.bq.json"
+    raw_table_name = table_name.replace("_external", "")
```

Comment:
Ah this is a little inconvenient that it uses the raw table name in the filename but now the tables have `_external`, but it works